gcsio/src/main/java/com/google/cloud/hadoop/gcsio/CreateObjectOptions.java

```diff
@@ -27,16 +27,14 @@
 @AutoValue
 public abstract class CreateObjectOptions {
 
-  public static final String CONTENT_TYPE_DEFAULT = "application/octet-stream";
-
   public static final CreateObjectOptions DEFAULT_NO_OVERWRITE = builder().build();
   public static final CreateObjectOptions DEFAULT_OVERWRITE =
       builder().setOverwriteExisting(true).build();
 
   public static Builder builder() {
     return new AutoValue_CreateObjectOptions.Builder()
         .setContentEncoding(null)
-        .setContentType(CONTENT_TYPE_DEFAULT)
+        .setContentType("application/octet-stream")
```

Comment:
why is this change required?

Reply:
There is a little utility in having default constants for each option class filed, if we already have default instance of the option class itself.